docs/guide/docs/channels/telegram.md

```diff
@@ -21,4 +34,4 @@ title: Telegram
 
 2. Restart Botpress and talk to your Telegram bot.
 
-> **Tip**: Don't forget to set `enabled: true` in your config. Otherwise the module will be disabled for your bot.
+> **Tip**: Don't forget to set `enabled: true` in your config. Otherwise, the module will be disabled for your Botpress chatbot.
```

Comment:
Good work using Botpress Chatbot, makes it clear we are different from Telegram.